### PR TITLE
Rewrite in go: Pack DEB

### PR DESCRIPTION
**Pull request by dokshina**

### CODE REVIEW

File: common/utils.go [hunk 1]
```diff
@@ -200,6 +201,26 @@ func FileLinesScanner(file *os.File) *bufio.Scanner {
...
+// GetFileContent returns file content
+func GetFileContent(path string) (string, error) {
+	file, err := os.Open(path)
+	defer file.Close()
+
+	if err != nil {
+		return "", err
+	}
+
+	fileContent := ""
```
Comment (olegrok) [inline common/utils.go:213]: I think `var fileContent string` is more appropriate

File: common/utils.go [hunk 2]
```diff
@@ -272,3 +293,14 @@ func WriteTgzArchive(srcDirPath string, destFilePath string) error {
 
 	return nil
 }
+
+func GetNextMajorVersion(version string) (string, error) {
+	parts := strings.Split(version, ".")
```
Comment (olegrok) [inline common/utils.go:298]: https://golang.org/pkg/strings/#SplitN

SplitN(1) should be enough

File: pack/deb.go [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+	log "github.com/sirupsen/logrus"
+	"github.com/tarantool/cartridge-cli/project"
+)
+
+var (
+	debianBinaryTemplate = templates.FileTemplate{
+		Path:    debianBinaryFileName,
+		Mode:    0644,
+		Content: "2.0\n",
```
Comment (olegrok) [inline pack/deb.go:19]: Do you really need "\n"?

File: pack/systemd_dir.go [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+
+		StateboardName:       projectCtx.StateboardName,
+		StateboardWorkDir:    filepath.Join("/var/lib/tarantool", projectCtx.StateboardName),
+		StateboardEntrypoint: project.StateboardEntrypointName,
+	}
+
+	if projectCtx.TarantoolIsEnterprise {
+		ctx.TarantoolDir = ctx.AppDir
+	} else {
+		ctx.TarantoolDir = "usr/bin" // TODO
```
Comment (olegrok) [inline pack/systemd_dir.go:73]: `/usr/bin/env tarantool`

Comment (dokshina) [reply pack/systemd_dir.go:73]: Decided to fix it later, now fixed to `/usr/bin`.

**Pull request merged**